Image build fails

**What happened:**
The image build workflow fails, one of the tests in the end does not pass.
`FAILED base/test/test_sgid_suid_files.py::test_sgid_suid_files[suid-whitelist_files1]`

**How to reproduce it**:
Run the test_sgid_suid_files chroot test.

--- a/features/base/test/test_sgid_suid_files.py
+++ b/features/base/test/test_sgid_suid_files.py
@@ -26,7 +26,7 @@ import pytest
                  "/usr/bin/gpasswd,root,root",
                  "/usr/bin/sudo,root,root",
                  "/usr/bin/passwd,root,root",
-                 "/usr/libexec/polkit-agent-helper-1,root,root",
+                 "/usr/lib/polkit-1/polkit-agent-helper-1,root,root",
                  "/usr/bin/pkexec,root,root"
                  ]
         )

--- a/features/vhost/test/test_sgid_suid_files.py
+++ b/features/vhost/test/test_sgid_suid_files.py
@@ -26,7 +26,7 @@ import pytest
                  "/usr/bin/gpasswd,root,root",
                  "/usr/bin/sudo,root,root",
                  "/usr/bin/passwd,root,root",
-                 "/usr/libexec/polkit-agent-helper-1,root,root",
+                 "/usr/lib/polkit-1/polkit-agent-helper-1,root,root",
                  "/usr/bin/pkexec,root,root"
                  ]
         )

